From 8b50aecee3cad40cb4b3c1a531f018cf649cdb00 Mon Sep 17 00:00:00 2001
From: Shammah Chancellor <shammah.chancellor@gmail.com>
Date: Sat, 12 Jun 2021 13:20:39 -0700
Subject: [PATCH 118/196] Update packaged filenames for gitian builds (#193)

Changing the CMake project name updates the filename of the
`ninja package_source` command and the resulting output filenames
of the gitian builds. This commit updates the package name to be in
line with the branding for lotus and to reduce confusion for users.
---
 CMakeLists.txt                              | 4 ++--
 contrib/gitian-descriptors/gitian-linux.yml | 4 ++--
 contrib/gitian-descriptors/gitian-osx.yml   | 4 ++--
 contrib/gitian-descriptors/gitian-win.yml   | 4 ++--
 doc/doxygen/Doxyfile.cmake.in               | 2 +-
 src/CMakeLists.txt                          | 2 +-
 src/config/CMakeLists.txt                   | 6 +++---
 src/qt/CMakeLists.txt                       | 8 ++++----
 8 files changed, 17 insertions(+), 17 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 4e4ba2543..84eb1bc7d 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -6,10 +6,10 @@ set(CMAKE_USER_MAKE_RULES_OVERRIDE
 	"${CMAKE_SOURCE_DIR}/cmake/modules/OverrideInitFlags.cmake"
 )
 
-project(bitcoin-abc
+project(lotus
 	VERSION 0.23.1
 	DESCRIPTION "Lotus is a full node implementation of the Lotus protocol."
-	HOMEPAGE_URL "https://www.bitcoinabc.org"
+	HOMEPAGE_URL "https://www.givelotus.org"
 )
 
 add_custom_target(print-version
diff --git a/contrib/gitian-descriptors/gitian-linux.yml b/contrib/gitian-descriptors/gitian-linux.yml
index 32dad3d74..b54f8082d 100644
--- a/contrib/gitian-descriptors/gitian-linux.yml
+++ b/contrib/gitian-descriptors/gitian-linux.yml
@@ -125,7 +125,7 @@ script: |
     -DCMAKE_TOOLCHAIN_FILE=${SOURCEDIR}/cmake/platforms/${CMAKE_TOOLCHAIN_FILE[${HOSTS[0]}]}
 
   ninja package_source
-  SOURCEDIST=`echo bitcoin-abc-*.tar.gz`
+  SOURCEDIST=`echo lotus-*.tar.gz`
   mv ${SOURCEDIST} ..
   popd
   DISTNAME=`echo ${SOURCEDIST} | sed 's/.tar.*//'`
@@ -133,7 +133,7 @@ script: |
   mkdir -p temp
   pushd temp
   tar -xf ../$SOURCEDIST
-  find bitcoin-abc-* | sort | tar --mtime="${REFERENCE_DATETIME}" --no-recursion --mode='u+rw,go+r-w,a+X' --owner=0 --group=0 -c -T - | gzip -9n > ../$SOURCEDIST
+  find lotus-* | sort | tar --mtime="${REFERENCE_DATETIME}" --no-recursion --mode='u+rw,go+r-w,a+X' --owner=0 --group=0 -c -T - | gzip -9n > ../$SOURCEDIST
   popd
 
   ORIGPATH="$PATH"
diff --git a/contrib/gitian-descriptors/gitian-osx.yml b/contrib/gitian-descriptors/gitian-osx.yml
index 7c1e65f62..72138dade 100644
--- a/contrib/gitian-descriptors/gitian-osx.yml
+++ b/contrib/gitian-descriptors/gitian-osx.yml
@@ -121,7 +121,7 @@ script: |
     -DCMAKE_TOOLCHAIN_FILE=${SOURCEDIR}/cmake/platforms/${CMAKE_TOOLCHAIN_FILE[${HOSTS[0]}]}
 
   ninja package_source
-  SOURCEDIST=`echo bitcoin-abc-*.tar.gz`
+  SOURCEDIST=`echo lotus-*.tar.gz`
   mv ${SOURCEDIST} ..
   popd
   DISTNAME=`echo ${SOURCEDIST} | sed 's/.tar.*//'`
@@ -130,7 +130,7 @@ script: |
   mkdir -p temp
   pushd temp
   tar -xf ../$SOURCEDIST
-  find bitcoin-abc-* | sort | tar --mtime="${REFERENCE_DATETIME}" --no-recursion --mode='u+rw,go+r-w,a+X' --owner=0 --group=0 -c -T - | gzip -9n > ../$SOURCEDIST
+  find lotus-* | sort | tar --mtime="${REFERENCE_DATETIME}" --no-recursion --mode='u+rw,go+r-w,a+X' --owner=0 --group=0 -c -T - | gzip -9n > ../$SOURCEDIST
   popd
 
   ORIGPATH="$PATH"
diff --git a/contrib/gitian-descriptors/gitian-win.yml b/contrib/gitian-descriptors/gitian-win.yml
index 3c5f74877..50e137d8d 100644
--- a/contrib/gitian-descriptors/gitian-win.yml
+++ b/contrib/gitian-descriptors/gitian-win.yml
@@ -136,7 +136,7 @@ script: |
     -DCMAKE_TOOLCHAIN_FILE=${SOURCEDIR}/cmake/platforms/${CMAKE_TOOLCHAIN_FILE[${HOSTS[0]}]}
 
   ninja package_source
-  SOURCEDIST=`echo bitcoin-abc-*.tar.gz`
+  SOURCEDIST=`echo lotus-*.tar.gz`
   mv ${SOURCEDIST} ..
   popd
   DISTNAME=`echo ${SOURCEDIST} | sed 's/.tar.*//'`
@@ -145,7 +145,7 @@ script: |
   mkdir -p temp
   pushd temp
   tar -xf ../$SOURCEDIST
-  find bitcoin-abc-* | sort | tar --mtime="${REFERENCE_DATETIME}" --no-recursion --mode='u+rw,go+r-w,a+X' --owner=0 --group=0 -c -T - | gzip -9n > ../$SOURCEDIST
+  find lotus-* | sort | tar --mtime="${REFERENCE_DATETIME}" --no-recursion --mode='u+rw,go+r-w,a+X' --owner=0 --group=0 -c -T - | gzip -9n > ../$SOURCEDIST
   mkdir -p $OUTDIR/src
   cp ../$SOURCEDIST $OUTDIR/src
   popd
diff --git a/doc/doxygen/Doxyfile.cmake.in b/doc/doxygen/Doxyfile.cmake.in
index 0d71440b1..10d435b61 100644
--- a/doc/doxygen/Doxyfile.cmake.in
+++ b/doc/doxygen/Doxyfile.cmake.in
@@ -38,7 +38,7 @@ PROJECT_NAME           = "${PACKAGE_NAME}"
 # could be handy for archiving the generated documentation or if some version
 # control system is used.
 
-PROJECT_NUMBER         = ${bitcoin-abc_VERSION}
+PROJECT_NUMBER         = ${lotus_VERSION}
 
 # Using the PROJECT_BRIEF tag one can provide an optional one line description
 # for a project that appears at the top of each page and should give viewer a
diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 057ff1867..2375392cf 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -1,6 +1,6 @@
 # Copyright (c) 2017 The Bitcoin developers
 
-project(lotusd)
+project(lotus-binaries)
 
 set(CMAKE_CXX_STANDARD 17)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
diff --git a/src/config/CMakeLists.txt b/src/config/CMakeLists.txt
index d7127f4e1..9b776d722 100644
--- a/src/config/CMakeLists.txt
+++ b/src/config/CMakeLists.txt
@@ -9,9 +9,9 @@ include(CheckCXXSymbolExists)
 include(CheckCXXSourceCompiles)
 
 # Version
-set(CLIENT_VERSION_MAJOR ${bitcoin-abc_VERSION_MAJOR})
-set(CLIENT_VERSION_MINOR ${bitcoin-abc_VERSION_MINOR})
-set(CLIENT_VERSION_REVISION ${bitcoin-abc_VERSION_PATCH})
+set(CLIENT_VERSION_MAJOR ${lotus_VERSION_MAJOR})
+set(CLIENT_VERSION_MINOR ${lotus_VERSION_MINOR})
+set(CLIENT_VERSION_REVISION ${lotus_VERSION_PATCH})
 
 option(CLIENT_VERSION_IS_RELEASE "Build a release version" OFF)
 
diff --git a/src/qt/CMakeLists.txt b/src/qt/CMakeLists.txt
index 5237e97a3..7f90c639e 100644
--- a/src/qt/CMakeLists.txt
+++ b/src/qt/CMakeLists.txt
@@ -383,12 +383,12 @@ if(${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
 		OUTPUT_NAME "${BITCOIN_QT_OSX_BUNDLE_NAME}"
 		MACOSX_BUNDLE_INFO_PLIST "${CMAKE_SOURCE_DIR}/share/qt/Info.plist.cmake.in"
 		MACOSX_BUNDLE_BUNDLE_NAME "${BITCOIN_QT_OSX_BUNDLE_NAME}"
-		MACOSX_BUNDLE_BUNDLE_VERSION "${bitcoin-abc_VERSION}"
+		MACOSX_BUNDLE_BUNDLE_VERSION "${lotus_VERSION}"
 		MACOSX_BUNDLE_GUI_IDENTIFIER "org.lotus.${BITCOIN_QT_OSX_BUNDLE_NAME}"
 		MACOSX_BUNDLE_ICON_FILE "${BITCOINQT_BUNDLE_ICON_NAME}"
-		MACOSX_BUNDLE_INFO_STRING "${bitcoin-abc_VERSION}, Copyright © 2009-${COPYRIGHT_YEAR} ${COPYRIGHT_HOLDERS_FINAL}"
-		MACOSX_BUNDLE_LONG_VERSION_STRING "${bitcoin-abc_VERSION}"
-		MACOSX_BUNDLE_SHORT_VERSION_STRING "${bitcoin-abc_VERSION}"
+		MACOSX_BUNDLE_INFO_STRING "${lotus_VERSION}, Copyright © 2009-${COPYRIGHT_YEAR} ${COPYRIGHT_HOLDERS_FINAL}"
+		MACOSX_BUNDLE_LONG_VERSION_STRING "${lotus_VERSION}"
+		MACOSX_BUNDLE_SHORT_VERSION_STRING "${lotus_VERSION}"
 		RESOURCE "${BITCOINQT_BUNDLE_RESOURCES}"
 	)
 	# The InfoPlist.strings files should be located in a resource subdirectory.
-- 
2.39.5 (Apple Git-154)

